### ISSUE TITLE
Android null pointer crash at splash screen

### ISSUE THREAD

**Issue by ddx-510** [wix/react-native-ui-lib#2875]
## Description

Using expo. When creating a release preview build, the Android app will crash at the splash screen.
When opening logcat at android studio, it points to **reactScreenMonitor.java** line 65 , null pointer issue

### Related to

- [x] Components
- [ ] Demo
- [ ] Docs
- [ ] Typings

### Steps to reproduce

1. open the app
2. crash at splash screen immediately

#### Expected behavior

Go inside the main page.

#### Actual behavior

Crash

### Code snippet

```
PID: 12852 java.lang.NullPointerException: 
Attempt to invoke virtual method 'android.view.View android.view.Window.getDecorView()' on a null object reference at com.wix.reactnativeuilib.keyboardinput.ReactScreenMonitor.registerWindowLayoutListener(ReactScreenMonitor.java:65) 
at com.wix.reactnativeuilib.keyboardinput.ReactScreenMonitor.onHostResume(ReactScreenMonitor.java:51) 
at com.facebook.react.bridge.ReactContext.onHostResume(ReactContext.java:308) 
at com.facebook.react.ReactInstanceManager.moveToResumedLifecycleState(ReactInstanceManager.java:778) 
at com.facebook.react.ReactInstanceManager.moveReactContextToCurrentLifecycleState(ReactInstanceManager.java:813) 
at com.facebook.react.ReactInstanceManager.-$$Nest$mmoveReactContextToCurrentLifecycleState(Unknown Source:0) 
at com.facebook.react.ReactInstanceManager$6.run(ReactInstanceManager.java:1202)
at android.os.Handler.handleCallback(Handler.java:958) at android.os.Handler.dispatchMessage(Handler.java:99) 
at android.os.Looper.loopOnce(Looper.java:222) at android.os.Looper.loop(Looper.java:314) 
at android.app.ActivityThread.main(ActivityThread.java:8530) at java.lang.reflect.Method.invoke(Native Method) 
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561) 
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1010)


```

### Screenshots/Video

![image](https://github.com/wix/react-native-ui-lib/assets/20592322/24a5cca8-d320-4800-93e8-1a3a260f4724)

### Environment

- React Native version: 0.72.6
- React Native UI Lib version: 7.13.0

### Affected platforms

- [x] Android
- [ ] iOS
- [ ] Web

**Comment by Svarto** [comment]
same issue

**Comment by ddx-510** [comment]
> same issue

I have forked and created my package (since maintainers have not approved my pr) and it fixed this bug.

If you need you can either check my pr #2878 and fix it by yourself, or you can use my package as a temporary solution :)
https://www.npmjs.com/package/@ddx0510/react-native-ui-lib

**Comment by asindhu** [comment]
Thanks folks... I believe I have this issue as well. Was incredibly time-consuming to debug because it is not reproducible for me in any emulator, only on an actual device. There were no stack trace, logs, or error info of any kind. Just once I got a null pointer exception similar to what's referenced above, after installing Sentry, but it was never clear whether this was related to my issue or not, since Sentry only captured it once out of the hundreds of times my app crashed.

I had to strip my app down to 0 and re-install modules one-by-one, fully rebuilding an Android bundle and installing it on a device on every change, to identify which of my modules was causing the problem -- indeed it was RNUILib.

Given that this issue crashes the app on startup and makes apps unusable on Android, hopefully it will be escalated and fixed soon. Has anyone had any luck reverting to an older version of RNUILib without the issue?

**Comment by asindhu** [comment]
> @Svarto I have forked and created my package (since maintainers have not approved my pr) and it fixed this bug.
> 
> If you need you can either check my pr #2878 and fix it by yourself, or you can use my package as a temporary solution :) https://www.npmjs.com/package/@ddx0510/react-native-ui-lib

This worked like a charm, thank you so much!!

**Comment by brahimjid** [comment]
> > @Svarto I have forked and created my package (since maintainers have not approved my pr) and it fixed this bug.
> > If you need you can either check my pr #2878 and fix it by yourself, or you can use my package as a temporary solution :) https://www.npmjs.com/package/@ddx0510/react-native-ui-lib
> 
> This worked like a charm, thank you so much!!



> > @Svarto I have forked and created my package (since maintainers have not approved my pr) and it fixed this bug.
> > If you need you can either check my pr #2878 and fix it by yourself, or you can use my package as a temporary solution :) https://www.npmjs.com/package/@ddx0510/react-native-ui-lib
> 
> This worked like a charm, thank you so much!!

are you using expo ?

**Comment by asindhu** [comment]
> > > @Svarto I have forked and created my package (since maintainers have not approved my pr) and it fixed this bug.
> > > If you need you can either check my pr #2878 and fix it by yourself, or you can use my package as a temporary solution :) https://www.npmjs.com/package/@ddx0510/react-native-ui-lib
> > 
> > 
> > This worked like a charm, thank you so much!!
> 
> > > @Svarto I have forked and created my package (since maintainers have not approved my pr) and it fixed this bug.
> > > If you need you can either check my pr #2878 and fix it by yourself, or you can use my package as a temporary solution :) https://www.npmjs.com/package/@ddx0510/react-native-ui-lib
> > 
> > 
> > This worked like a charm, thank you so much!!
> 
> are you using expo ?

Yes, using Expo and managed workflow.

**Comment by M-i-k-e-l** [comment]
Should be fixed by #2878
